frontend/src/metabase/modes/components/drill/AutomaticDashboardDrill.jsx

```diff
@@ -15,10 +16,17 @@ export default ({ question, clicked }: ClickActionProps): ClickAction[] => {
 
   // questions with a breakout
   const dimensions = (clicked && clicked.dimensions) || [];
+
+  // ExpressionDimensions don't work right now (see metabase#16680)
+  const includesExpressionDimensions = dimensions.some(dimension => {
+    return isExpressionField(dimension.column.field_ref);
+  });
+
   if (
     !clicked ||
     dimensions.length === 0 ||
-    !MetabaseSettings.get("enable-xrays")
+    !MetabaseSettings.get("enable-xrays") ||
+    includesExpressionDimensions
```

Comment:
Do these chained conditions mean something that can be named?

Might be nice to have them as a constant, with a declarative name.

Reply:
done